custom/conf/app.example.ini

```diff
@@ -295,9 +298,21 @@ SSH_KEY_TEST_PATH =
 ; Path to ssh-keygen, default is 'ssh-keygen' which means the shell is responsible for finding out which one to call.
 SSH_KEYGEN_PATH = ssh-keygen
 ; Enable SSH Authorized Key Backup when rewriting all keys, default is true
-SSH_BACKUP_AUTHORIZED_KEYS = true
+SSH_AUTHORIZED_KEYS_BACKUP = true
```

Comment:
Since the change, it's a break change.

Reply:
I was correcting the documentation as `SSH_BACKUP_AUTHORIZED_KEYS` isn't correct. (it has nothing to do with my change, I can remove it again if you like)

See https://github.com/go-gitea/gitea/blob/master/modules/setting/setting.go#L120 on master

Reply:
@lunny he's correct here: https://github.com/go-gitea/gitea/blob/95d478c29c60c38c17dcaadeda0192979ea0d451/modules/setting/setting.go#L722

It's been SSH_AUTHORIZED_KEYS_BACKUP for 3 years!